recipes/xtrabackuprb.rb

```diff
@@ -1,24 +1,30 @@
 include_recipe 'git'
 include_recipe 'percona::package_repo'
 
+if platform_family?('rhel') && node['platform_version'].to_i == 6
+  include_recipe 'yum-epel'
+  package 'libev'
+end
+
+package 'percona-xtrabackup'
```

Comment:
This installation fails since we now don't include postfix on the vms. I recommend you include the ``osl-postfix`` cookbook above to work around this.